sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -14,26 +14,51 @@ import { PollOperationState } from '@azure/core-lro';
 import { TokenCredential } from '@azure/core-auth';
 
 // @public
-export interface AnalysisPollOperationState<TResult> extends PollOperationState<TResult>, JobMetadata {
+export interface Actions {
+    extractKeyPhrasesActions?: ExtractKeyPhrasesAction[];
+    recognizeEntitiesActions?: RecognizeCategorizedEntitiesAction[];
+    recognizePiiEntitiesActions?: RecognizePiiEntitiesAction[];
 }
 
 // @public
-export interface AnalyzeBatchTasksResult {
+export interface AnalysisPollOperationState<TResult> extends PollOperationState<TResult>, OperationMetadata {
+}
+
+// @public
+export interface AnalyzeBatchActionsOperationMetadata extends OperationMetadata {
+    displayName?: string;
+    failedActionsCount?: number;
```

Comment:
do we still need the `count` as suffix? @maririos

Reply:
I prefer to keep Count to be clear about the meaning of the number we return.

Reply:
Also, we should explore `actionInTotal` property which indicate the total counts of all actions. Even user can sum up the failed + inProgress + succeeded actions but since swagger has the value, we should explore it as well to save the user's calculation.

Reply:
I did the renamings but kept the count postfix.